docs/functions/Bundles/Start-UPloadVCFBundle.md

```diff
@@ -1,38 +0,0 @@
-# Start-UploadVCFBundle
-
-### Synopsis
-The Start-UploadVCFBundle starts upload of bundle to SDDC Manager
-
-### Syntax
-```
-Start-UploadVCFBundle -json <path to json file>
```

Comment:
Module Name should be 'Start-VCFUploadBundle'

Reply:
It's a deleted file